app/models/orchestration_stack_retire_request.rb

```diff
@@ -0,0 +1,14 @@
+class OrchestrationStackRetireRequest < MiqRequest
```

Comment:
@d-m-u Would it make sense to add an intermediate class called MiqRetireRequest and put all the common code in there and these this classes can just provide the TASK_DESCRIPTION and SOURCE_CLASS_NAME, since they seem to be repeating the same boiler plate code in all the child Request classes.